sci2zero/TeslaRIS-frontend
# Feature/assessment

## Description

Added initial assessment model as well as basic initial CRUD for all entities. Added view and download statistic views to test the soundness of the implementation. Fixed some bugs.

## How should this be tested?

This should be tested in pair with [backend PR](https://github.com/sci2zero/TeslaRIS-backend/pull/109). When logged in as an administrator user, you should be able to perform basic CRUD operations on assessment entities. Statistics should be viewable with any access level.

## Checklist
